docs/en_US/builtin_assessor.rst

```diff
@@ -1,6 +1,16 @@
 Builtin-Assessors
 =================
 
+In order to save our computing resources, NNI supports an early stop policy and creates **Assessor** to finish this job.
```

Comment:
Sometimes to save computing resources, we need to stop the under performance trials earlier. NNI supports early stop mechanism with built-in as well as self defined **Assessor** in the way similar to Tuners.